src/routes/sequencing/+layout.svelte

```diff
@@ -1,27 +1,83 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  import { page } from '$app/stores';
+  import ChevronRightIcon from '@nasa-jpl/stellar/icons/chevron_right.svg?component';
   import PhoenixIcon from '../../assets/aerie-phoenix-logo.svg?component';
   import Nav from '../../components/app/Nav.svelte';
   import CssGrid from '../../components/ui/CssGrid.svelte';
+  import { SearchParameters } from '../../enums/searchParameters';
+  import { workspaces } from '../../stores/sequencing';
+  import { getSearchParameterNumber } from '../../utilities/generic';
   import type { PageData } from './$types';
 
   export let data: PageData;
+
+  let workspaceId: number | null = null;
+
+  $: workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID, $page.url.searchParams);
+
+  $: workspace = $workspaces.find(workspace => workspace.id === workspaceId);
```

Comment:
Same thing here, I think we should define variables before assigning them reactivity.